### PR TITLE
Added to social-apps.md: tumblr, pinterest, exercism, steam, ancestry.

### LINKED ISSUE #29: [FEATURE] Add Social Media names having public profile feature
# Add Social Media names having public profile feature

## Description
We've a list of social media website/app names which has public profile feature.
Like Instagram has: `https://instagram.com/@<username>`, Github has: `https://github.com/<username>`.

The complete list is provided in [`social-apps.md`](https://github.com/Param302/SocialRepo/blob/main/.github/docs/social-apps.md).
Your task is find and add more social media app/website names in [`social-apps.md`](https://github.com/Param302/SocialRepo/blob/main/.github/docs/social-apps.md).

### Requirements
- Make sure the name of app/website you are adding must have a public profile link feature.
- Make sure the name should not be present in the list already.
- You can add new names as well, like `twitter` changed to `X`, so both `twitter` and `X` will be treated differently.
- Do the Pull Request once you've collected minimum of `5` new social media/apps which are not already present in the list.

If you have any suggestion or idea, you're welcome to share.

_Let's make SocialRepo's universal for all Social media spaces_ 🙌

### PR DESCRIPTION
**Pull request by kimchiiboiii**
# Title
Added to social-apps.md: tumblr, pinterest, exercism, steam, ancestry

## Description
Added 5 new social media websites with public profiles.
Profile URLs:
1. Steam: https://steamcommunity.com/id/**your-profile-url-here**/
2. Ancestry: https://www.ancestry.com/profile/**your-profile-url-here**
3. Tumblr: https://www.tumblr.com/**your-profile-url-here**
4. Pinterest: https://www.pinterest.com/**your-profile-url-here**/
5. Exercism: https://exercism.org/profiles/**your-profile-url-here**

---
## Issue Ticket Number
Fixes #29

---
## Type of change
- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

---
# Checklist:
- [X] I have followed the contributing guidelines of this project as mentioned in [CONTRIBUTING](./docs/CONTRIBUTING)
- [X] I have checked to ensure there aren't other open [Pull Requests](https://github.com/Param302/SocialRepo/pulls) for the same update/change?
- [X] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes needed to the documentation

### CODE REVIEW

File: .github/docs/social-apps.md [hunk 1]
```diff
@@ -11,22 +11,27 @@ This file contains the list of social app/website names which has public user pr
...
 
 ### Miscellaneous
 
 -   [x] Airchat
+-   [x] Ancestry
 -   [x] Bio.link
 -   [ ] Codechef
 -   [x] Codeforces
 -   [x] Dev.to
+-   [x] Exercism
```
Comment (Param302) [inline .github/docs/social-apps.md:34]: Just add the name of the app/website with `[ ]`, no need to check the checkbox, the checkbox is for some other purpose.

**Review by Param302 — CHANGES_REQUESTED** [review]
Hey @kimchiiboiii ,
First of all thanks for your contribution!


Can you tell what is Ancestry and How it's public profile useful?

And
Please do the changes and let me know about Ancestry.

Also, make sure to follow basic guidelines, by discussion and asking to assigned the issue before working on.

As you've already raised the PR, and seeing your effort I will assign you the issue #29  . Please comment on the same so that I can assign you.

_Happy Contributing!_

**Comment by kimchiiboiii** [comment]
Hey, thanks so much.

Ancestry is a site where you can take a DNA test to get your ethnicity results, and map out your family tree, meet other users who have DNA matches with you.
However now that I think about it, there isn't much socializing besides the private messages and User Search feature.
Maybe at most you would just be messaging a distant cousin who has a DNA match with you, on that website.

So I think it's better for me to go ahead and replace it with Mastodon which is more used and definitely is a social media site.

**Review by Param302 — APPROVED** [review]
Hey @kimchiiboiii ,

Thanks for your valuable contribution to **SocialRepo**.

I am merging it to main branch

Looking forward for more contributions. :D
_Keep Contributing!_ 🙌

**Pull request merged**